content/community/community.md

```diff
@@ -3,11 +3,47 @@ title: Community
 description: The TiKV community
 ---
 
-TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to [contribute here](/docs/3.0/reference/contribute/).
+TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to contribute [here](/community/contribute/).
 
-The TiKV community is organised into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is below, follow the links for more information.
+## SIGs
+
+The TiKV community is organized into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is listed below. follow the links for more information.
```

Comment:
"The current roster of SIGs is listed below." This is not quite right, prefer either "The current roster of SIGs is below." or "The current SIGs are listed below."